### DISCUSSION TITLE
Максим будет дальнейшее развитие Lathecode. И где можно найти полный перечень реализованных команд на текущее время.

### DISCUSSION THREAD

**Discussion by Viktor1972-avi** [kachurovskiy/nanoels#209] [General]
Lathecode

**Comment by kachurovskiy** [comment]
Привет Виктор, пока желания дальше развивать у меня нет - но поскольку весь код в доступе с документацией то я надеюсь что кто-то перехватит знамя :)

Полная документация по коммандам lathecode - https://github.com/kachurovskiy/lathecode

**Reply by Viktor1972-avi** [reply]
Добрый вечер. Максим в Грамматике есть inside = "INSIDE" comment, что это означает? Нет не в одном примере. Это для внутреннего точения?

**Reply by kachurovskiy** [reply]
Да, для внутреннего точения - но пока это не поддерживается полностью поэтому нет в документации. https://github.com/kachurovskiy/lathecode/issues/3
